### DISCUSSION TITLE
Linguist didn't detect languages in repository

### DISCUSSION THREAD

**Discussion by instinxt** [github-linguist/linguist#5400] [Classification] [answered]
Following are the two repositories
https://github.com/instinxt/Tinder-clone
https://github.com/instinxt/Tiktok-clone

---
The languages weren't detected on these repositories

All i did was remote push and set the master as default git base branch , whilst deleting the main branch

**Comment by lildude** [comment, marked as answer]
Looks like to job that does the analysis failed when you pushed. Pushing another change will kick off the job again.
